R/tranform_assay.R

```diff
@@ -131,23 +131,26 @@ transform_utility  = function(input_read_RNA_assay, transform_fx, external_path,
   # Find the mode (peak) value of the counts
   mode_value <- density_est$x[which.max(density_est$y)]
   
-  # If the mode value is negative, shift counts to make the mode zero
+  # If the mode value is negative, shift counts and counts used for estimation to make the mode zero
   if (mode_value < 0) {
     counts <- counts + abs(mode_value)
-  }
```

Comment:
You could have added 

```r
counts_light_for_checks = counts_light_for_checks + abs(mode_value)
```
At line 137, rather than increasing the complexity of the code (with if else, and creating a new variable). 


As discussed with Julie, what could be the learning lesson @multimeric and @jiskcoder, for applying this king of thinking in the future?